corehq/apps/reports/views.py

```diff
@@ -1,12 +1,14 @@
 from __future__ import absolute_import
-from copy import copy
+import copy
```

Comment:
Looks like this change broke the saving new reports on production https://sentry.io/dimagi/commcarehq/issues/444236551/.

Looks like an easy fix, but curious what prompted changing this import and whether there's something you think I need to be careful about.

Reply:
Assume it was just so you could use `copy.deepcopy` below; maybe your IDE unwisely made this change without your knowledge.

Reply:
It was to use `copy.deepcopy`. Can't blame the editor, though, just my mistake. Should be the easy fix (updating the two other usages of `copy` in this file).